### ISSUE TITLE
Release 1.14.0

### ISSUE THREAD

**Issue by enricoros** [enricoros/big-AGI#417]
## Release checklist:

- [x] Create a new [Release Issue](https://github.com/enricoros/big-AGI/issues/new?assignees=enricoros&projects=enricoros/4&template=maintainers-release.md&title=Release+1.14.0)
  - [x] Replace 1.13.0 with the _former_ release, and _1.14.0_ with THIS
- [x] Update the [Roadmap](https://github.com/users/enricoros/projects/4/views/2) calling out shipped features
- [x] Create and update a [Milestone](https://github.com/enricoros/big-agi/milestones) for the release
  - [x] Assign this task
  - [x] Assign all the shipped roadmap Issues
  - [x] Assign the relevant [recently closed Isssues](https://github.com/enricoros/big-agi/issues?q=is%3Aclosed+sort%3Aupdated-desc)
- Code changes:
  - [x] Create a release branch 'release-x.y.z': `git checkout -b release-1.14.0`
  - [x] Create a temporary tag `git tag v1.14.0 && git push opensource --tags`
  - [x] Create a [New Draft GitHub Release](https://github.com/enricoros/big-agi/releases/new), and generate the automated changelog (for new contributors)
  - [x] Update the release version in package.json, and `npm i`
  - [x] Update in-app News [src/apps/news/news.data.tsx](/src/apps/news/news.data.tsx)
  - [x] Update the in-app News version number
  - [x] Update the README.md with the new release
  - [x] Copy the highlights to the [docs/changelog.md](/docs/changelog.md)
- Release:
  - [x] merge onto main `git checkout main && git merge --no-ff release-1.14.0`
  - [x] re-tag `git tag -f v1.14.0 && git push opensource --tags -f`
  - [x] verify deployment on Vercel
  - [x] verify container on GitHub Packages
  - [x] update the GitHub release
  - [x] push as stable `git push opensource main:main-stable`
- Announce:
  - [x] Discord announcement
  - [x] Twitter announcement

### Links

- Milestone: https://github.com/enricoros/big-AGI/milestone/14
- GitHub release: https://github.com/enricoros/big-AGI/releases/tag/v1.14.0
- Former release task: #405 

## Artifacts Generation

```markdown
You help me generate the following collateral for the new release of my opensource application called big-AGI. The new release is 1.14.0.
To familiarize yourself with the application, the following are the Website and the GitHub README.md.
```

- paste the URL: https://big-agi.com
- drag & drop: [README.md](https://raw.githubusercontent.com/enricoros/big-AGI/main/README.md)

```markdown
I am announcing a new version, 1.14.0.
For reference, the following was the collateral for 1.13.0 (Discord announcement, GitHub Release, in-app-news file news.data.tsx).
```

- paste the former: `discord announcement`,
- `GitHub release`,
- `news.data.tsx`,
- `changelog.md`

```markdown
The following are the new developments for 1.14.0:

- ...
- git log --pretty=format:"%h %an %B" v1.13.0..v1.14.0 | clip
```

- paste the link to the milestone (closed) and each individual issue (content will be downloaded)
- paste the output of the git log command

### news.data.tsx

```markdown
I need the following from you:

1. a table summarizing all the new features in 1.14.0 with the following columns: 4 words description (exactly what it is), short description, usefulness (what it does for the user), significance, link to the issue number (not the commit)), which will be used for the artifacts later
2. then double-check the git log to see if there are any features of significance that are not in the table
3. then score each feature in terms of importance for users (1-10), relative impact of the feature (1-10, where 10 applies to the broadest user base), and novelty and uniqueness (1-10, where 10 is truly unique and novel from what exists already) 
4. then improve the table, in decreasing order of importance for features, fixing any detail that's missing, in particular check if there are commits of significance from a user or developer point of view, which are not contained in the table
5. then I want you then to update the news.data.tsx for the new release
```

### Readme (and Changelog)

```markdown
I need you to update the README.md and the with the new release.
Attaching the in-app news, with my language for you to improve on, but keep the tone.
```

### GitHub release

```markdown
Please create the 1.14.0 Release Notes for GitHub, following the format of the 1.13.0 GitHub release notes attached before.
Use a truthful and honest tone, understanding that people's time and attention span is short.
Today is 2024-XXXX-YYYY.
```

Now paste-attachment the former release notes (or 1.5.0, which was accurate and great), including the new contributors and
some stats (# of commits, etc.), and roll it for the new release.

### Discord announcement

```markdown
Can you generate my 1.14.0 big-AGI discord announcement from the GitHub Release announcement?
Please keep the formatting and style of the discord announcement for 1.13.0, but with the new messaging above.
```

**Comment by enricoros** [comment]
Partial Release Notes:
 - Gemini: support for 1.5 models, and the new list format
 - Improved search and folder experience
 -

**Comment by enricoros** [comment]
Screenshots
---
Improve Latex rendering:
![image](https://github.com/enricoros/big-AGI/assets/32999/112780d6-4bfa-4bfe-b5d0-040e27c2e5b7)

**Comment by enricoros** [comment]
news.data.tsx
---
  {
    versionCode: '1.14.0',
    versionName: 'Modelmorphic',
    versionCoverImage: coverV114,
    versionDate: new Date('2024-03-06T08:00:00Z'),
    items: [
      { text: <><B issue={407}>Perplexity</B> support, including Online models</>, issue: 407, icon: PerplexityIcon },
      { text: <><B issue={427}>Groq</B> support, with speeds up to 500 tok/s</>, issue: 427, icon: GroqIcon },
      { text: <>Support for new Mistral-Large models</>, icon: MistralIcon },
      { text: <>Support for Google Gemini 1.5 models and various improvements</>, icon: GoogleIcon as any },
      { text: <>Deeper LocalAI integration including support for <B issue={411}>model galleries</B></>, icon: LocalAIIcon },
      { text: <>Major <B href='https://twitter.com/enricoros/status/1756553038293303434'>performance optimizations</B>: runs faster, saves power, saves memory</> },
      { text: <>Improvements: auto-size charts, search and folder experience</> },
      { text: <>Perfect chat scaling, with rapid keyboard shortcuts</> },
      { text: <>Also: diagrams auto-resize, open code with StackBlitz and JSFiddle, quick model visibility toggle, open links externally, docs on the web</> },
      { text: <>Fixes: standalone LaTeX blocks, close views by dragging, knowledge cutoff dates, crashes on Google translate (thanks dad)</> },
    ],
  },

**Comment by enricoros** [comment]
README.md
---
### What's New in 1.14.0 · March 6, 2024 · Modelmorphic

- New **[Perplexity](https://www.perplexity.ai/)** and **[Groq](https://groq.com/)** integration (thanks @Penagwin). [#407](https://github.com/enricoros/big-AGI/issues/407), [#427](https://github.com/enricoros/big-AGI/issues/427)
- **[LocalAI](https://localai.io/models/)** deep integration, including support for [model galleries](https://github.com/enricoros/big-AGI/issues/411)
- **Mistral** Large and Google **Gemini 1.5** support
- Performance optimizations: runs [much faster](https://twitter.com/enricoros/status/1756553038293303434?utm_source=localhost:3000&utm_medium=big-agi), saves lots of power, reduces memory usage
- Enhanced UX with auto-sizing charts, refined search and folder functionalities, perfected scaling
- And with more UI improvements, documentation, bug fixes (20 tickets), and developer enhancements
- [Release notes](https://github.com/enricoros/big-AGI/releases/tag/v1.14.0), and changes [v1.13.1...v1.14.0](https://github.com/enricoros/big-AGI/compare/v1.13.1...v1.14.0) (233 commits, 8,000+ lines changed)

**Comment by enricoros** [comment]
GitHub release:
---
# 🚀 big-AGI 1.14.0 - Modelmorphic

### The new shape of intelligence and performance.

We're releasing big-AGI 1.14.0, "Modelmorphic." This update expands models support, focuses on improving performance, and delivers user experience enhancements. With over 230 commits and over 8,000 lines of code changed, it's designed to make big-AGI smarter, faster, more reliable.

![release-cover-v1 14 0 b](https://github.com/enricoros/big-AGI/assets/32999/0763d3c0-d165-4978-b45f-37c6e4b79b41)

- Added support for **Perplexity**, including Sonar Online models, and **Groq**, including models with up to 500 tok/s speeds. Contribution by @Penagwin.
- Added **Mistral** Large models, and Google **Gemini 1.5** models.
- Deeper **LocalAI** integration with support for model galleries and keys.  
- Performance **Optimizations**: comprehensive improvements for faster operation, lower power consumption, and better memory use  
- UI and Usability: updated navigation, search, scaling, shortcuts, and chat interactions for a smoother experience.
- Addressed issues with image warnings, standalone LaTeX blocks, hidden personas in the selector, and more.
  
### For Developers:
- Customizable PlantUML Server via an environment variable, thanks to @fredliubojin.
- New customization documentation and analytics support are now included.
- Added /info/debug for better diagnostics.
- Introduced a "controller-based" chat architecture to simplify per-chat data management, including ephemerals.
- Full changelog: https://github.com/enricoros/big-AGI/compare/v1.13.1...v1.14.0

### New Contributors

Special thanks to our new contributors @Penagwin and @jacksongoode. @Penagwin work has been instrumental for making this release a success.

## In Summary

Version 1.14.0 - **Modelmorphic** represents a leap forward in our commitment to providing advanced, reliable, and user-centric AI tools. We continue to eschew fluff in favor of substance, focusing on what truly matters to our users and developers.

Explore the new features and improvements _and upgrade to Modelmorphic today_, available via code, Docker, or directly on [big-agi.com](https://big-agi.com).

Join our AGI community on [Discord](https://discord.gg/MkH4qj2Jp9) and get involved with us! Your feedback and contributions are what drive us forward. And now enjoy `Modelmorphic` 🚀

**Comment by enricoros** [comment]
Discord Announcement
---
@everyone :sparkles: **big-AGI 1.14.0 - Modelmorphic** Launched

:partyblob~1: **The new era of AI performance & integration.** :partyblob~1:

"**Modelmorphic**" turbocharges big-AGI with enhanced model support, performance leaps, and UX refinements. Over 230 commits and 8,000 lines of code have shaped this release.

- **Perplexity & Groq** models now onboard, thanks to @Penagwin's brilliance. Experience online llms and up to 500 tok/s speeds!
- **Mistral Large** and **Gemini 1.5** models are here
- **LocalAI** digs deeper with model galleries and key integrations
- **Performance Optimizations**: We've cranked up the efficiency dial - expect lightning-fast operations, minimal power usage, and optimized memory
- **UI/UX Polish**: Navigating, searching, scaling, and chatting just got smoother and more intuitive
- **Fixes Galore**: From image warnings to LaTeX quirks, we've ironed out the creases for a seamless experience

**For the Tech-Savvy:**
- Say hello to customizable **PlantUML Servers** (hat tip to @fredliubojin).
- Dive into our fresh **customization docs** and **analytics support**.
- Debug with ease using **/info/debug**.
- Embrace our new "controller-based" chat architecture for streamlined data management.

:rocket: Enjoy **Modelmorphic** - available on https://get.big-agi.com, and on Docker, on GitHub, and you know the drill - and our little fella also does! :rocket:
